clusterConfigSchema.yml

```diff
@@ -15,3 +15,7 @@ clusters:
       node: ""
       bmc_user: "required if physical or bf"
       bmc_password: "required if physical or bf"
+    # optionally describe additional node attributes
+    hosts:
+    - name: "A"
+      images_path: "required path to the VM pool directory"
```

Comment:
It's a bit confusing that we say that this is optional, but the string contains the word "required". I would say that this is optional too in the string

Reply:
Or maybe /path/to/pool/directory